python/hail/expr/functions.py

```diff
@@ -1065,6 +1065,40 @@ def or_missing(predicate, value):
     predicate = to_expr(predicate)
     return _func("orMissing", predicate._type, predicate, value)
 
+@typecheck(x=expr_int32, n=expr_int32, p=expr_numeric,
+           alternative=enumeration("two.sided", "greater", "less"))
+@args_to_expr
+def binom_test(x, n, p, alternative):
+    """Performs an exact binomial test on `x` with `n` trials at probability `p`.
```

Comment:
I'd remove the description below and just put this here:
Returns the `exact binomial test <https://en.wikipedia.org/wiki/Binomial_test>`__  p-value for probability `p` of success, given `x` successes in `n` trials.

Reply:
Too long. The first description should be under 80 chars, for sure.